examples/tutorials/Part 12 bis - Encrypted Training on MNIST.ipynb

```diff
@@ -0,0 +1,664 @@
+{
```

Comment:
<p>&nbsp;et -&gt; and (using all my vast knowledge in french I inferred that)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2411/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Haha</p>